test/test_point.py

```diff
@@ -107,6 +108,31 @@ def test_point_degrees_are_normalized(self):
         self.assertEqual((-85, -175, 375), tuple(point))
         point = Point(-95, -185, 375)
         self.assertEqual((85, 175, 375), tuple(point))
+        point = Point(-0.0, -0.0, 375)
+        self.assertEqual((0.0, 0.0, 375.0), tuple(point))  # note that the zeros might be negative
+        # ensure that negative zeros are normalized to the positive ones
+        self.assertEqual((1.0, 1.0, 1.0), tuple(math.copysign(1.0, x) for x in point))
+        point = Point(90, 180, 375)
+        self.assertEqual((90, 180, 375), tuple(point))
+        point = Point(-90, -180, 375)
+        self.assertEqual((-90, -180, 375), tuple(point))
+        point = Point(-270, -540, 375)
+        self.assertEqual((-90, -180, 375), tuple(point))
+        initial = (90, 180)
```

Comment:
> Should we also include 90 and 180 in normalization to be consistent ie make abs(latitude) >= 90 and abs(longitude) >= 180?

No, I'd rather not introduce such a change right now without further analyzing. Let's keep it as is for now.

> What do you think of a tests, was it too much, or just fine?

Above this line it looks great to me. But the loops below seem to be a bit complex. If `Point(-270, -540, 375)` or `Point(270, 540, 375)` test would fail, then anything in the range for `counter ` from 2 to 10 would also fail. And vice-versa: if any of the tests in the range from 2 to 10 would fail, then these two tests should fail too, because there's no such logic in the code which affects behaviour for the values which `abs` is more than 90/180.

Thereby, IMO it would be enough to simply add a `Point(270, 540, 375)` test instead of the loops below. And then it should be ready to go 😉

Reply:
>No, I'd rather not introduce such a change right now without further analyzing. Let's keep it as is for now.

Okay.

>Above this line it looks great to me. But the loops below seem to be a bit complex. If Point(-270, -540, 375) or Point(270, 540, 375) test would fail, then anything in the range for counter from 2 to 10 would also fail. And vice-versa: if any of the tests in the range from 2 to 10 would fail, then these two tests should fail too, because there's no such logic in the code which affects behaviour for the values which abs is more than 90/180.

I agree, I just thought that 90 + 180*N and 180 + 360*N (for N=1..inf) was a prompt to implement it.

>Thereby, IMO it would be enough to simply add a Point(270, 540, 375) test instead of the loops below. And then it should be ready to go wink

Added! It was a long run, but definitely not the last ;)